src/js/totalRelaying.js

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("floe.chartAuthoring.totalRelaying", {
+        gradeNames: ["fluid.modelComponent", "autoInit"],
+        model: {
+            total: {
+                // value: number,
+                // percentage: number
+            },
+            dataSet: {
+                // in the style of the dataSet in dataEntryPanel or pie/legend
```

Comment:
Can we improve this comment so other ppl don't need to look into dataEntryPanel or pie/legend to understand its structure?